tests/test_standalone.py

```
@@ -0,0 +1,184 @@
+"""
+Standalone test for _filter_and_renormalize_other function.
+This test can run without installing dependencies.
+"""
+
+from typing import Dict, List
```

Ruff (0.14.10)

`typing.Dict` is deprecated, use `dict` instead

(UP035)